pycbc/waveform/waveform.py

```diff
@@ -116,6 +117,24 @@ def _lalsim_td_waveform(**p):
                float(p['delta_t']), float(p['f_lower']), float(p['f_ref']),
                lal_pars,
                _lalsim_enum[p['approximant']])
+    except:
```

Comment:
I'm trying to think whether I like this or not. Do you think you could add a 'fail tolerant' boolean though? (with default to True).